src/components/code/CodeTabs.jsx

```
@@ -0,0 +1,237 @@
+                onClick={() => handleTabChange(index)}
+                className={clsx(
+                  'relative whitespace-nowrap border-b-2 px-3 py-2 text-sm font-medium transition-colors',
+                  isActive
+                    ? 'border-accent-500 text-accent-600 dark:text-accent-400'
+                    : 'border-transparent text-slate-600 hover:text-slate-900 dark:text-slate-400 dark:hover:text-slate-200'
+                )}
+                role="tab"
+                aria-selected={isActive}
+                aria-controls={`code-panel-${tab.tabId}`}
+                id={`tab-${tab.tabId}`}
+              >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)